KAA/README.md

```diff
@@ -0,0 +1,116 @@
+# Set up account and device on KaaIoT cloud platform
+This detailed instruction will walk you through the process of creating your own iot device in just a few steps.
+
+## What is KaaIoT
+KaaIoT is a cloud service provider allowing for device management, data collection and its visualization. You can create powerful dashboards in a matter of seconds. It's free of charge with a limit of maximum 5 connected devices.
+
+## Requirements
+
+### Python 3, PIP 
+Python can be downloaded from this [website](https://www.python.org/downloads) (in case of using [Anaconda environment](https://www.anaconda.com/products/individual), please skip this part and refer to "**Installation**" section). The Python version required for this project is: either 3.6 or 3.7 (preferably). Please follow installation instructions from their website. 
```

Comment:
It's already there. Requirements is a bigget section containing only Python in this case.
"in case of using [Anaconda environment] (...) skip this part and refer to (...)" because it is not needed in this case.